docs/src/pages/customization/components/components.md

```diff
@@ -44,7 +44,7 @@ Using the dev tools, now you know that you need to target the `.MuiSlider-thumb`
 
 ### Overriding styles with class names
 
-If you would like to override the styles of the components using classes, you can use the `className` prop available on each component. For overriding the styles of the slots inside the component, you may use the global classes available for each slot, as described in the previous section.
+If you would like to override the styles of the components using classes, you can use the `className` prop available on each component. For overriding the styles of the slots inside the component, you can use the global classes available for each slot, as described in the previous section.
```

Comment:
So this is where it gets confusing. A "slot" is just a placeholder where you plug in some component to fill the gap, right? So you can't style a slot, as that's like styling an empty seat in a car, as opposed to the person sitting in it.

(Or I'm still misunderstanding the "slots" terminology)

Reply:
This sounds about right, the slot should likely refer to the placeholder, slots are occupied by components.

Reply:
Maybe we should restructure to:

> To override the styles of different elements in your component

or

> To override the styles of different parts of your component

Reply:
Updated with https://github.com/mui-org/material-ui/pull/24016/commits/4b75ea0af6a5381d07318ee718861687eb494d6b